libs/vscode-ui/components/src/lib/autocomplete/autocomplete.component.html

```diff
@@ -1,3 +1,38 @@
-<datalist id="{{ field.name }}_enum">
-  <option *ngFor="let value of field.enum" [value]="value"></option>
-</datalist>
+<div class="description">
+  {{ field.description }}
+</div>
+
+<div class="modified-indicator"></div>
+
+<div class="control">
+  <div class="autocomplete-container">
+    <input
+      #textInput
+      [formControl]="control"
+      class="text-input"
+      type="text"
+      role="textbox"
+      [required]="field.required"
+      [attr.list]="field.enum ? field.name + '_enum' : undefined"
```

Comment:
Do we need this binding? We aren't using data lists anymore and so far as I've read this attribute is meant to hook up an input to a datalist

Reply:
`list` binding removed